### PR TITLE
Update README.md by cha-warawita

### LINKED ISSUE #3: - [x] Observed for testing is noted as an enhancement
- [x] Observed for testing
- [x] Can be closed

_Originally posted by @charitha-warawita in https://github.com/srilankantrendz/ConsoleAppIQO2/issues/2#issuecomment-1790180508_

### PR DESCRIPTION
**Pull request by charitha-warawita**
Update README.md by cha-warawita

### CODE REVIEW

File: README.md
Comment (srilankantrendz) [inline README.md:1]: Looks ok. only change is the v1.2 => 1.3 change. approved.

**Comment by charitha-warawita** [comment]
I think this is a basic version change only. please approve. -- c-w.

**Comment by srilankantrendz** [comment]
> I think this is a basic version change only. please approve. -- c-w.

I'll provide one rec to do. wait...

**Review by srilankantrendz — CHANGES_REQUESTED** [review]
change v1.3 to v1.4 and re-commit. [todo]

**Comment by charitha-warawita** [comment]
> change v1.3 to v1.4 and re-commit. [todo]

ok. noted will do in 5 mins.

**Review by charitha-warawita — COMMENTED** [review]
req code change is done. pls approve.

- [x ] done

**Review by srilankantrendz — APPROVED** [review]
OK. **Approved**.

**Pull request merged**